PyFlyt/core/aviary.py

```diff
@@ -79,9 +81,9 @@ def reset(self, seed: None | int = None):
         # reset the camera position to a sane place
         self.resetDebugVisualizerCamera(
             cameraDistance=5,
-            cameraYaw=30,
-            cameraPitch=-30,
-            cameraTargetPosition=[0, 0, 1],
+            cameraYaw=0,
+            cameraPitch=-50,
+            cameraTargetPosition=[0, 0, 0],
```

Comment:
If possible, turn this into kwargs in the init and default them to the old values. (Remember to use typehints)

Reply:
Will revert this original, `fixedwing.py` resets the camera to track the plane cameraman style, so this is irrelevant for `fixedwing.py`